src/deployment/base/BaseDeployment.s.sol

```diff
@@ -49,7 +49,8 @@ abstract contract BaseDeployment is Test {
         bool envMode = vm.envBool(modeKey);
         MODE = envMode ? Mode.BROADCAST : Mode.SIMULATE;
 
-        bytes32 envNetwork = keccak256(abi.encodePacked(vm.envString(networkKey)));
+        string memory nw = vm.envString(networkKey);
+        bytes32 envNetwork = keccak256(abi.encodePacked(nw));
```

Comment:
I think better naming here would be `string memory envNetwork` and `bytes32 envNetworkHash`.